R/bar_plot_emission_profile.R

```diff
@@ -59,10 +59,9 @@ bar_plot_emission_profile <- function(data,
 #' @noRd
 calc_benchmark_emission_profile <- function(data, risk_var, benchmarks) {
   data <- data |>
-    mutate(risk_category_var = as_risk_category(data[[risk_var]])) |>
     filter(.data$benchmark %in% benchmarks) |>
     group_by(.data$risk_category_var, .data$benchmark) |>
-    summarize(count = n()) |>
+    summarise(count = n()) |>
     group_by(.data$benchmark) |>
     mutate(proportion = .data$count / sum(.data$count))
   return(data)
```

Comment:
It's best to reserve return() for early returns. Maybe a refactoring for another PR?

Reply:
Yes! I'll create another ticket for it :D !